storage/blob.go

```diff
@@ -473,44 +459,29 @@ func (b BlobStorageClient) ContainerExists(name string) (bool, error) {
 }
 
 // SetContainerPermissions sets up container permissions as per https://msdn.microsoft.com/en-us/library/azure/dd179391.aspx
-func (b BlobStorageClient) SetContainerPermissions(container string, containerPermissions ContainerPermissions) (err error) {
+func (b BlobStorageClient) SetContainerPermissions(container string, containerPermissions ContainerPermissions, timeout int, leaseID string) (err error) {
```

Comment:
Timeout is already an `int` (or a `uint`) in a bunch of places. I also suspect some operations that support the timeout are not including it in their parameters. We can fix this, but I think it is better in another PR.